metabase/metabase
# add husky with pre-commit lint hook

### Description

Add [husky](https://github.com/typicode/husky) with `pre-commit` hook that runs prettier on staged filed.

How to test:
Try to commit files with invalid formatting and ensure prettier has been executed
